### PR TITLE
added favicon, fixed hyphens, fixed line break

### LINKED ISSUE #55: Create Favicon for website

### LINKED ISSUE #82: Home Page clean up
- [x] separate home page content from layouts sections
- [x] hyphening seems too aggressive. Remove or reduce

### PR DESCRIPTION
**Pull request by designedbinary**
fixes #82 
fixes #55 

Added favicon using https://realfavicongenerator.net/ but the results were a little blurry, particularly on my computer. Need to test it more. 

removed hyphening. 

fixed some headlines so that they do not create orphans

### CODE REVIEW

File: docs/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (conorgil) [inline docs/site.webmanifest]: I'm not familiar with this file. What is it for? Any links for me to read up?

**Comment by conorgil** [comment]
@designedbinary I forgot to check for a changelog entry. The changes are pretty minimal, but best to be consistent and throw them in there.

File: docs/site.webmanifest (hunk 1 above)
Comment (designedbinary) [reply docs/site.webmanifest]: https://realfavicongenerator.net/ I'm not sure what this is either. Got lazy and just C/P'd everything.

**Comment by designedbinary** [comment]
@conorgil Ok, updated the changelog. Will merge

**Pull request merged**